docker-compose.test.yml

```
@@ -3,15 +3,12 @@ services:
       dockerfile: Dockerfile
-    command: ["sh", "/app/scripts/start-prod.sh"]
+      target: production
     restart: unless-stopped
-    volumes:
-      - .:/app
-      - /app/node_modules
     ports:
       - '3001:3001'
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://${WEBSITE_DOMAIN}:${NEXT_PUBLIC_PROD_PORT}"]
+      test: [ "CMD", "curl", "-f", "http://${WEBSITE_DOMAIN}:${NEXT_PUBLIC_PROD_PORT}" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)